docs/01-start.md

```diff
@@ -48,9 +48,16 @@ conda install -c apple tensorflow-deps
 # Manually build tensorflow-text until a collaborator build is available.
 # This was tested using clang version 15 - you may get non-working wheels with earlier versions of clang.
 mkdir ~/builds && git clone https://github.com/tensorflow/text.git ~/builds/text
-cd ~/builds/text && git checkout 6064f1bf8fd078777b6c8690986b908c28764a94
+cd ~/builds/text && git checkout 0f9f6df5b4da19bc7a734ba05fc4fa12bccbedbe
+# Patch tensorflow-text to support tf 2.16.1.
+git pull origin pull/1273/head -X ours
```

Comment:
Thanks to @jiya-zhang's tip, if we install TF manually before trying to build `tensorflow-text`, it will not attempt to install TF again.

```suggestion
pip install tensorflow==2.16.1
cd ~/builds/text && git checkout 0f9f6df5b4da19bc7a734ba05fc4fa12bccbedbe
```

Reply:
Done.